vendor/os-sunnyvalley/pkg-descr

```diff
@@ -0,0 +1,20 @@
+Sensei is a plugin for networking software complementing them with
+state of the art next generation network security features, such as
```

Comment:
Imho: This plugin adds a proprietary repository to install Sensei, which is...


Just to say more clearly what it really does.